app/poi_api/service.py

```diff
@@ -0,0 +1,65 @@
+from flask import jsonify
+from typing import List, Dict
+from .poi_suggestion import POI_suggestion
+from .poi import POI
+from .errors import POINotFoundError, InvalidPOISuggestionError
+from app import common
+from datetime import datetime
+from app.firebase import firestore_db, POI_COLLECTION, POI_PROPOSAL_COLLECTION
+
+poi_collection = firestore_db.collection(POI_COLLECTION)
+poi_proposal_collection = firestore_db.collection(POI_PROPOSAL_COLLECTION)

```

Comment:
This fucks with testing cause you force it to create a firebase client when you import the file instead of when you actually end up calling the function that need to call firebase. Can you wrap these in a function instead?